### PR TITLE
cmd/snap: add client test support methods

### PR DESCRIPTION
**Pull request by zyga**
Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

**Comment by chipaca** [comment]
:+1:

File: cmd/snap/main_test.go [hunk 1]
```diff
@@ -62,3 +65,26 @@ func (s *SnapSuite) Stdout() string {
...
+func DecodedRequestBody(r *http.Request, c *C) map[string]interface{} {
+	var body map[string]interface{}
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&body)
+	c.Assert(err, IsNil)
+	return body
+}
+
+// EncodeResponseBody writes JSON-serialized body to the response writer.
+func EncodeResponseBody(w http.ResponseWriter, c *C, body interface{}) {
```
Comment (pedronis) [inline cmd/snap/main_test.go]: c *C as 2nd argument is strange, any particular reasons?

seems more common to be uniform and put first or last

Comment (zyga) [inline cmd/snap/main_test.go]: No particular reason, which way would you prefer it then?

Comment (pedronis) [inline cmd/snap/main_test.go]: @zyga as far as I can tell from grepping the typical style already for helpers taking *check.C is first (as I also tend to do personally), so both need changing

**Comment by pedronis** [comment]
thanks, +1

**Pull request merged**